code/modules/supermatter/supermatter.dm

```diff
@@ -12,6 +12,8 @@
 #define WARNING_DELAY 30 		//seconds between warnings.
 #define AUDIO_WARNING_DELAY 30
 
+var/global/total_supermatter_objects = 0
```

Comment:
You could make this a static variable on `/obj/machinery/power/supermatter`, functionally the same but it's faster and you don't have to fuck with the global helper (!)

Reply:
And if you don't you shouldn't tag the variable as `/global`

Reply:
>it's faster

Is it? Where did you hear that?

I agree that a static variable makes more sense though

Reply:
I was under the impression statics were the fastest, followed by globals

Reply:
http://www.byond.com/forum/?post=2140538

Reply:
Oh, and while this does make more sense, the comment about the global helper doesn't apply anymore

Reply:
I cannot even *begin* to imagine what is going on internally that makes static vars faster than regular datum vars, but that's BYOND for you

Reply:
Apparently, when you read a static var on an object, you're actually reading the static var of the *type* the var holding that object is *cast* as
So that explains that I guess